code/game/machinery/botlaunchpad.dm

```diff
@@ -0,0 +1,82 @@
+/obj/machinery/botpad
+	name = "Bot pad"
+	desc = "A lighter version of the orbital mech pad modified to launch bots. Requires linking to a remote to function."
+	icon = 'icons/obj/telescience.dmi'
+	icon_state = "botpad"
+	circuit = /obj/item/circuitboard/machine/botpad
+	// ID of the console, used for linking up
+	var/id = "botlauncher"
+	var/obj/item/botpad_remote/connected_remote
+	var/mob/living/simple_animal/bot/launched_bot // we need this to recall the bot
+
+/obj/machinery/botpad/Destroy()
+	if(connected_remote)
+		connected_remote.connected_botpad = null
+		connected_remote = null
+	return ..()
+
+/obj/machinery/botpad/screwdriver_act(mob/user, obj/item/tool)
+	. = ..()
+	if(!.)
+		return default_deconstruction_screwdriver(user, "botpad-open", "botpad", tool)
+
+/obj/machinery/botpad/crowbar_act(mob/user, obj/item/tool)
+	..()
```

Comment:
this doesn't do anything

Reply:
This still shouldn't be calling the parent. Your build is falling cause you're working off an outdated branch, not cause you didn't call the parent.